frontend/pages/_app.js

```diff
@@ -2,19 +2,7 @@ import App from 'next/app'
 import Head from 'next/head'
 import { Message } from 'semantic-ui-react'
 
-import 'semantic-ui-css/components/reset.css'
-import 'semantic-ui-css/components/site.css'
-
-import 'semantic-ui-css/components/button.css'
-import 'semantic-ui-css/components/container.css'
-import 'semantic-ui-css/components/divider.css'
-import 'semantic-ui-css/components/grid.css'
-import 'semantic-ui-css/components/icon.css'
-import 'semantic-ui-css/components/image.css'
-import 'semantic-ui-css/components/input.css'
-import 'semantic-ui-css/components/menu.css'
-import 'semantic-ui-css/components/message.css'
-import 'semantic-ui-css/components/popup.css'
+import 'semantic-ui-css/semantic.min.css'
```

Comment:
I'd like to keep importing these individually. I know it's a pain but it does reduce the size of the CSS.

Reply:
It's on my Todo though I have to figure out what to import actually :sweat_smile: